java-rest-service/accelerator.yaml

```diff
@@ -80,8 +80,25 @@ accelerator:
       defaultValue: "Manage customer profiles"
       dependsOn:
         name: exposeOpenAPIEndpoint
+    - name: updateBoot3
+      inputType: checkbox
+      dataType: boolean
+      label: Use Spring Boot 3.0 (Requires Java 17)
+      defaultValue: false
+    # override java version choices, Java 8 is not available for this sample
+    - name: javaVersion
+      inputType: select
+      label: Java version to use
+      choices:
+      - value: "11"
+        text: Java 11
+      - value: "17"
+        text: Java 17
+      defaultValue: "11"
   imports:
   - name: java-version
+    expose:
```

Comment:
We recently added support for exposing no options using the syntax:
```
expose: []
```

Reply:
ahh, nice